cmd/podman/system/prune.go

```diff
@@ -0,0 +1,103 @@
+package system
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podman/registry"
+	"github.com/containers/libpod/cmd/podman/utils"
+	"github.com/containers/libpod/cmd/podman/validate"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	pruneOptions = entities.SystemPruneOptions{}
+
+	pruneDescription = fmt.Sprintf(`
+	podman system prune
+
+        Remove unused data
+`)
+
+	pruneCommand = &cobra.Command{
+		Use:     "prune [flags]",
+		Short:   "Remove unused data",
+		Args:    validate.NoArgs,
+		Long:    pruneDescription,
+		RunE:    prune,
+		Example: `podman system prune`,
+	}
+	force bool
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: pruneCommand,
+		Parent:  systemCmd,
+	})
+	flags := pruneCommand.Flags()
+	flags.BoolVarP(&force, "force", "f", false, "Do not prompt for confirmation.  The default is false")
+	flags.BoolVarP(&pruneOptions.All, "all", "a", false, "Remove all unused data")
```

Comment:
this might be a @mheon question, should this be "unused data" or "unused data and objects"?